src/components/Button/Button.tsx

```diff
@@ -6,9 +6,9 @@ import styles from "./Button.module.css";
 
 type Props = ButtonProps;
 
-/** 
- * Кнопка. 
- */
+/*
+    Кнопка
```

Comment:
Неверный формат комментария
```suggestion
/**
 * Кнопка
 */
```